v2/bugsnag_test.go

```diff
@@ -98,7 +84,16 @@ func TestNotify(t *testing.T) {
 
 	md := MetaData{"test": {"password": "sneaky", "value": "able", "broken": complex(1, 2), "recurse": recurse}}
 	user := User{Id: "123", Name: "Conrad", Email: "me@cirw.in"}
-	config := generateSampleConfig(ts.URL)
+
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
I'd like to see a little more about this in the PR description to help for future readers – and me now :) – on why this change was made.

Reply:
Tried to describe it better in PR :)